### ISSUE TITLE
NuSLERP breaks

### ISSUE THREAD

**Issue by RedrixHD** [arcee-ai/mergekit#472]
Greetings!
I've been testing the new merge method NuSLERP. It always returns this, no matter the config:
```
Input should be a valid dictionary or instance of GatherTensors [type=model_type, input_value=PermutedEmbeddings(gather...ride_architecture=None)), input_type=PermutedEmbeddings]
```
My OS is Fedora Linux. I've updated all the dependencies. This seems to be an issue with the code of NuSLERP. I've tried to check the source code, and can't find why, how and where it creates PermutedEmbeddings instead of GatherTensors (isn't PermutedEmbeddings part of GatherTensors in the source code?). But I'm a novice at this, so I don't know.

**Comment by cg123** [comment]
Thanks for the report! This should be fixed with #473 - please let me know if you run into any further trouble.

**Comment by RedrixHD** [comment]
Thanks! It works: https://huggingface.co/redrix/matricide-12B-Unslop-Unleashed-v2
